### DISCUSSION TITLE
paper/linear-algebra

### DISCUSSION THREAD

**Discussion by giscus** [ysw421/blog#12] [General]
# paper/linear-algebra

Hello, i am siwon :)

https://www.siwonsw.com/paper/linear-algebra

**Comment by dongkyu12345** [comment]
제 롤모델은 시원님입니다!!

**Comment by 0720al** [comment]
시원님 정말 '시원'명쾌하게 정리를 잘 해놓으셨네요~~ 도움이 많이 되었습니다!!!!

**Comment by swimoneshot** [comment]
시원님은 정말 최고입니다
